libcontainerd/types_windows.go

```diff
@@ -67,6 +67,11 @@ type CredentialsOption struct {
 	Credentials string
 }
 
+// DNSOption is a CreateOption that provides container specific DNS settings
```

Comment:
Could/should lthis be combined with the existing `NetworkEndpointsOption` to avoid more proliferations to this pattern of side-band options? Not a problem if it doesn't make sense, just asking the question :innocent: